models/depth_camera/depth_camera.sdf.jinja

```diff
@@ -26,18 +26,18 @@
       </visual>
       <sensor name="depth_camera" type="depth">
         {%- if ros_version == '2' -%}
-        <always_on>0</always_on>
-        <update_rate>30</update_rate>
+        <always_on>1</always_on>
```

Comment:
do you really want to have it always on ? What was controlling that parameter before ?

Reply:
We want it to be streaming images all the time - essentially to be the same as the camera.
To tell the truth, I'm not actually 100% sure what the behaviour is when `always_on` is disabled.